sql/item_basic.sql

```diff
@@ -4458,12 +4458,12 @@ INSERT INTO `item_basic` VALUES (4722,312,'scroll_of_enfire_ii','enfire_ii',1,34
 INSERT INTO `item_basic` VALUES (4723,313,'scroll_of_enblizzard_ii','enblizzard_ii',1,34444,28,0,924);
 INSERT INTO `item_basic` VALUES (4724,314,'scroll_of_enaero_ii','enaero_ii',1,34444,28,0,874);
 INSERT INTO `item_basic` VALUES (4725,315,'scroll_of_enstone_ii','enstone_ii',1,34444,28,0,884);
-INSERT INTO `item_basic` VALUES (4726,316,'scroll_of_enthunder_ii','enthunder_ii',1,34444,28,0,874);
+INSERT INTO `item_basic` VALUES (4726,316,'scroll_of_enthunder_ii','enthunder_ii',1,34444,28,0,800);
 INSERT INTO `item_basic` VALUES (4727,317,'scroll_of_enwater_ii','enwater_ii',1,34444,28,0,1000);
 INSERT INTO `item_basic` VALUES (4728,120,'scroll_of_teleport-yhoat','teleport-yhoat',1,1676,28,0,1642);
 INSERT INTO `item_basic` VALUES (4729,121,'scroll_of_teleport-altep','teleport-altep',1,1676,28,0,1642);
-INSERT INTO `item_basic` VALUES (4730,122,'scroll_of_teleport-holla','teleport-holla',1,1676,28,0,1556);
-INSERT INTO `item_basic` VALUES (4731,123,'scroll_of_teleport-dem','teleport-dem',1,1676,28,0,1556);
+INSERT INTO `item_basic` VALUES (4730,122,'scroll_of_teleport-holla','teleport-holla',1,1676,28,0,2100);
+INSERT INTO `item_basic` VALUES (4731,123,'scroll_of_teleport-dem','teleport-dem',1,1676,28,0,2100);
 INSERT INTO `item_basic` VALUES (4732,124,'scroll_of_teleport-mea','teleport-mea',1,1676,28,0,1556);
```

Comment:
Mea is 98% likely to be the same price as the others, but if it hasn't be _officially_ confirmed, I'm okay leaving the row as is